Description: Do not use the sphinx_rtd_theme in conf.py
 Upstream is using a weird sphinx theme not available in Debian. So we just
 fix the issue by selecting default as html theme.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-05-28

--- python-wrapt-1.8.0.orig/docs/conf.py
+++ python-wrapt-1.8.0/docs/conf.py
@@ -95,13 +95,7 @@ pygments_style = 'sphinx'
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
-if on_rtd:
-    html_theme = 'default'
-else:
-    import sphinx_rtd_theme
-    html_theme = 'sphinx_rtd_theme'
-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
